sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -604,7 +592,7 @@ namespace Azure { namespace Data { namespace Tables {
      * @brief Upsert Entity result.
      *
      */
-    struct UpsertEntityResult final : public MergeEntityResult, UpdateEntityResult, AddEntityResult
+    struct UpsertEntityResult final : public UpdateEntityResult, AddEntityResult
```

Comment:
Unrelated to this PR, but this inheritance on the returned response model seems a bit odd.
FWIW, I don't fully understand the design/motivation, so take it with a grain of salt (and maybe it's fine), but it's not common for us to have such object hierarchy.

Reply:
done